backends/test/suite/runner.py

```diff
@@ -9,6 +9,14 @@
 
 import torch
 
+# Set of unsupported ops that should cause tests to be skipped
+UNSUPPORTED_PORTABLE_OPS = {
```

Comment:
If we are adding a Portable flow, how would these show up there? Some `PTE_FAIL`?

Reply:
Fair question. They won't be reported as failures. I think that's mostly fine, since the main goal is to validate backends. That being said, we could only skip these tests for flows that are expected to delegate - though backend op libs might make it trickier. I'll make this change.